contrib/intel/jenkins/build.py

```diff
@@ -31,11 +31,12 @@ def build_libfabric(libfab_install_path, mode, cluster=None):
     else:
         prov_list = common.default_prov_list
 
-    if ('ucx' in libfab_install_path):
-        config_cmd.append('--enable-ucx=yes')
-
     for prov in prov_list:
-       config_cmd.append(f'--enable-{prov}={enable_prov_val}')
+       if (prov == 'ucx'):
```

Comment:
why is this not an 'and'?

Reply:
> why is this not an 'and'?
it is to avoid the case when ucx flag is false but 
for loop iterates on prov_list which will lands in the else case 
for ucx and still enable it.